### PR TITLE
Status Card now working with influx data

### PR DESCRIPTION
**Pull request by upatricck**
## Done
Changed the status card to use the data in influx, and also fixed a bug that was causing the information to be from only one site

### CODE REVIEW

**Comment by upatricck** [comment]
Closed issue #160

File: seshdash/data/db/influx.py [hunk 1]
```diff
@@ -269,8 +269,43 @@ def get_latest_measurement_point_site(self, site, measurement_name, database=Non
...
+        query = "SELECT * FROM %s WHERE site_name='%s' LIMIT 1" % (measurements, site.site_name)
+        points = list(self._influx_client.query(query,database=db).get_points())
+
+        # Creating a dictionary with measurement as key and point as value
+        measurement_dict = {}
+        for i, measurement in enumerate(measurement_list):
+            try:
+                measurement_dict[measurement] = points[i]
+            except:
+                print "No measurements found for %s" % measurement
```
Comment (jlark) [inline seshdash/data/db/influx.py]: please don't use print : )

File: seshdash/views.py [hunk 2]
```diff
@@ -669,20 +669,27 @@ def silence_alert(request):
 
 @login_required
 def get_latest_bom_data(request):
```
Comment (jlark) [inline seshdash/views.py]: let's make this general ass discussed.

File: seshdash/data/db/influx.py [hunk 3]
```diff
@@ -269,8 +269,43 @@ def get_latest_measurement_point_site(self, site, measurement_name, database=Non
...
+        
+        # Handling the db to be used
+        db = self.db
+        if database:
+           db = database        
+
+        measurements = ""
+       
+        # Formatting the measurement list
+        for i, measurement in enumerate(measurement_list):
```
Comment (jlark) [inline seshdash/data/db/influx.py]: let's write less code `",".join(measurement_list)`

**Comment by jlark** [comment]
#TODO Sort out how the measurement units are handled.

**Pull request merged**